tests/utils/isIOSOlderThan10.test.js

```diff
@@ -1,5 +1,5 @@
 /* global assert, suite, test */
-var isIOSOlderThan10 = require('utils/isIOSOlderThan10');
+var isIOSOlderThan10 = require('utils/isIOSOlderThan10.js');
```

Comment:
You missed this one? `src/utils/isIOSOlderThan10.js` is not converted either.

Reply:
Actually it can be removed. It's only referenced in this test and never exposed in the bundle. This check seems to originally be used to workaround a bug in the webvr-polyfill on Safari.

Reply:
Talking about webvr-polyfill we removed. I think we should also remove the cardboardModeEnabled flag in xr-mode-ui component and documentation.